### PR TITLE
Fix typo and improve readability

### PR DESCRIPTION
**Pull request by eust-w**
* Rename updatAvailableMenuID to updateAvailableMenuID
* Replace unused cmd parameter with _ in RunServer function
* Fix typos in comments

### CODE REVIEW

File: api/client.go [hunk 1]
```diff
@@ -303,7 +303,7 @@ func (c *Client) List(ctx context.Context) (*ListResponse, error) {
 	return &lr, nil
 }
 
-// List running models.
+// ListRunning list running models.
```
Comment (jmorganca) [inline api/client.go:306]:
```suggestion
// ListRunning lists running models.
```

**Comment by jmorganca** [comment]
Small comment otherwise LGTM

File: api/client.go (hunk 1 above)
Comment (eust-w) [reply api/client.go]: SGTM

**Comment by eust-w** [comment]
PTAL @jmorganca

**Review by jmorganca — APPROVED** [review]
Thank you!

**Pull request merged**